src/core/system_impl.h

```diff
@@ -261,6 +261,11 @@ class SystemImpl : public Sender {
 
     double timeout_s() const;
 
+    void set_server_armed(bool armed);
+    bool is_server_armed() const;
+    void set_custom_mode(uint32_t custom_mode);
+    uint32_t get_custom_mode() const;
```

Comment:
Maybe add a comment above: these are for the case where mavsdk is used as an autopilot.